dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessors.java

```diff
@@ -12,15 +12,34 @@
 public class DbParameterAccessors implements Iterable<DbParameterAccessor> {
```

Comment:
Good question... I'm not sure we can cay that a parameter is kind of `Attribute`. In `OracleEnvironment` we have some nested classes named like `DbParameterOrColumnInfo `, `DbDictionaryParamsOrColumnsIterator`, etc.

Technically `DbParameterAccessor` is used for accessing parameters of database queries (sp calls, insert, update).

I'm adding a note about that in #483.